packages/flutter/lib/src/rendering/stack.dart

```diff
@@ -103,10 +103,24 @@ class RelativeRect {
   }
 
   /// Convert this [RelativeRect] to a [Rect], in the coordinate space of the container.
+  ///
+  /// See also:
+  ///
+  ///  * [toSize], which returns the size part of the rect, based on the size of
+  ///    the container.
   Rect toRect(Rect container) {
     return new Rect.fromLTRB(left, top, container.width - right, container.height - bottom);
```

Comment:
I'm confused by this.  On the face of it, I would have expected the return value to be:

  `new Rect.fromLTRB(left, top, left + container.width - right, top + container.height - bottom);

Reply:
RelativeRect gives the distance from the edges of the container. Rect gives the distance from the top left of the container.